bambi/models.py

```diff
@@ -802,6 +803,8 @@ def predict(
         inplace=True,
         include_group_specific=True,
         sample_new_groups=False,
+        num_draws=None,
+        random_seed=42,
```

Comment:
I would make this `None` by default, which means the random seed is determined by the random number generator (usually using something like time).